src/components/Tasks/Sidebar.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+
+import InboxIcon from '@material-ui/icons/Inbox';
+import LabelIcon from '@material-ui/icons/LabelOutlined';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { setSettings } from '../../store/actions';
+import { StoreState } from '../../store';
+
+import { List, ListItem, ListSubheader } from '../../widgets/List';
+
+const useStyles = makeStyles({
+  root: {
+    '& .MuiListItemIcon-root': {
```

Comment:
Why did you have to do that? Rely on internal class names like this is a bad idea and a sure way to get breaks down the line.
You can just use the `rightIcon` property and put whatever you want there, e.g:
`rightIcon={<Badge>{count}</Badge}` or even `rightIcon={<span>{count}</span>}`.

Edit: I see you already did it excatly this way, so why the extra classes? What's failing? Maybe it's something we can fix in the underlying code.

Reply:
it's to deal with these ugly styles. This is how it looks without my custom styles. See how the number just floats there? I want to right-align it some way.
<img width="342" alt="Screen Shot 2020-03-17 at 2 55 42 AM" src="https://user-images.githubusercontent.com/12586676/76844748-fcdc4380-67fa-11ea-84fd-0dd41f5370ca.png">

I think part of the reason I have to deal with it like this is that the MUI ListItem API doesn't really provide a way to have some secondary, right-aligned icon. The closest is `ListItemSecondaryAction`, but using that seems semantically incorrect.

Could we modify our `ListItem` to accommodate right-aligning the `rightIcon`?

Reply:
You just need to set the following on the span:
`<span style={{ width: '100%', textAlign: 'right' }}>{amount}</span>` and it should work. No need to hack it from the outside. :)

Reply:
Nice, that is much simpler than what I had

Reply:
It's not a huge deal, but I was also trying to reduce the space between the left icon and the text. Just seems a little too wide to me. Do you have any suggestions for how I could handle that?

Reply:
I wouldn't touch that. It's part of the theme and how the library does lists.

Reply:
Ok I'll just leave it as it